packages/create-cloudflare/src/__tests__/git.test.ts

```diff
@@ -147,9 +147,8 @@ describe("git helpers", () => {
 
 	describe("offerGit", async () => {
 		test("happy path", async () => {
-			const ctx = createTestContext();
+			const ctx = createTestContext("test", { projectName: "test" });
```

Comment:
The `testCreateContext()` helper sets up the `ctx.args` to be all the `C3_DEFAULT_ARGS` values, which undermines the idea that the user has not provided any command line args.  By providing an args object here we ensure that none of the CLI args are set so that we have control over them in the tests.

Reply:
Should your comment be a code comment?

Reply:
9442311fdea8dfd51738da4b6d541919e1d46c2f